test_app/schema.py

```diff
@@ -56,8 +53,7 @@ def get_best_friend(self):
         return self.data.friends.order_by('pk').first()
 
 
-@schema.register_query_root
-class QueryRoot(Object):
+class QueryRoot(BaseQueryRoot):
```

Comment:
Why can we not define `schema = Schema(QueryRoot)` in this file so that each test doesn't need to define its own Schema in the same way?